specs/bin/stalePullRequestSpec.js

```diff
@@ -117,38 +145,35 @@ describe('stalePullRequest', function () {
             .catch(done.fail);
     });
 
-    it('stalePullRequest._processPullRequest posts expected message for secondary stale pull request', function (done) {
+    it('stalePullRequest._processPullRequest does not post when asked to stop', function (done) {
         var repositorySettings = new RepositorySettings();
-        var commentsUrl = 'commentsUrl';
+        var commentsUrl = 'https://url';
         var pullRequest = {
             comments_url: commentsUrl
         };
 
         spyOn(requestPromise, 'get').and.callFake(function (options) {
-            if (options.url === commentsUrl) {
+            if (options.resolveWithFullResponse === true) {
+                return Promise.resolve({headers: {link: '<https://url?page=2>; rel="next",<https://url?page=3>; rel="last"'}});
+            }
+            else if (options.url === commentsUrl + '?page=3') {
```

Comment:
should be formatted `} else if (...) {`

Reply:
Ah, turns out there was multiple places where it was like that!